SPARQLWrapper/SmartWrapper.py

```diff
@@ -169,7 +168,7 @@ def __contains__(self, key):
         :return: whether there is a binding of the variable in the return
         :rtype: Boolean
         """
-        if len(self.bindings) == 0:
+        if len(self.bindings) == 0 or self.variables is None:
```

Comment:
Looks like this can go into a separate PR.

Reply:
934e334